.github/workflows/main.yml

```diff
@@ -12,17 +12,17 @@ jobs:
   build_and_test:
     strategy:
       matrix:
-        node: ['12', '14', '16']
+        node: ['14', '16']
```

Comment:
For my own information, did we remove Node v12 due to a package that was upgraded no longer supporting Node v12? I think that's fine, just a few things we should keep in mind:
- Stencil still supports Node v12 (until Stencil v3, which is coming soon) - this probably isn't much of an issue 
- We should update `engines` [in package.json](https://github.com/stencil-community/stencil-eslint/pull/85/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L56) to state we support v14+ (since we don't test against 12 anymore)

Reply:
You are right. I will close this PR for now. I managed to keep supporting Node 12, by keeping the same version of Jest, used on Stencil Core. Thanks for the info. 🙏